labware-library/src/components/labware-ui/labels.ts

```
@@ -11,14 +8,15 @@ export const getWellLabel = (spec: LabelSpec, fallback?: LabelSpec): string =>
   getLabel(WELL_TYPE_BY_CATEGORY, spec, fallback)
 
 function getLabel(
-  labelMap: { [string]: string },
+  labelMap: Record<string, string>,
   spec: LabelSpec,
   fallback?: LabelSpec
 ): string {
   let { displayCategory } = spec.metadata
 
-  if (spec.groups && spec.groups.length > 0) {
+  if ('groups' in spec && spec.groups.length > 0) {
     const categories = spec.groups.map(g => g.metadata.displayCategory)
+    // @ts-expect-error(IL, 2021-03-25): `uniqBy` 2nd arg should be optional, but TS wants it explicit
```

We could merge in a fixed up declaration by adding a `typings/lodash.d.ts` rather than ignoring the error

```ts
// typings/lodash.d.ts
import 'lodash/uniqBy'

declare module 'lodash/uniqBy' {
  function uniqBy<T>(array: List<T> | null | undefined, iteratee?: ValueIteratee<T>): T[];
  export = uniqBy
}

---

I wonder if we have the latest lodash type defs?

---

discussed in chat, seems like we need to wait for defs to be fixed